zulipterminal/model.py

```diff
@@ -979,6 +1017,10 @@ def get_user_info(self, user_id: int) -> Optional[TidiedUserInfo]:
         else:
             user_role = raw_user_role
 
+        custom_profile_data = api_user_data.get("profile_data", {})
+        cleaned_custom_profile_data = self._clean_custom_profile_data(
+            custom_profile_data
+        )
```

Comment:
This change and the TidiedUserInfo change in an earlier commit can go together? The big new function above should go into an earlier commit, with tests.

I understand this works for getting things working :)